test/test_mps.py

```diff
@@ -1493,6 +1493,122 @@ def test_exp(self, device="mps", dtype=torch.float):
             a = torch.tensor(v, dtype=dtype, device="mps") * b
             self.compare_with_numpy(torch.exp, np.exp, a)
 
+    # TODO: I would like to only have one copy of this test code and run it on
+    # both CUDA and MPS, but I'm not sure where is a good place to put it so
+    # that both test_torch and test_mps can have access to it. I guess
+    # probably somewhere in `torch.testing`
```

Comment:
Since we don't have a test file for copy, and this is broader than MPS specifically. I would create a `test_lazy_clone.py` to hold this (and other tests related to lazy copy /COW behaviors). 

IMO adding a new file here would be better than burying the test somewhere that requires extra work to find it/remember where it lives.

Reply:
Makes sense, I'll do that!